packages/nextjs/pages/[contractAddress]/[network].tsx

```diff
@@ -168,6 +169,7 @@ const ContractDetailPage = ({ addressFromUrl, chainIdFromUrl }: ServerSideProps)
           )}
         </div>
       </div>
+      <SwitchTheme className="fixed bottom-3 right-3 z-50" />
```

Comment:
When there is scrollbar, it overlaps the scroll. Maybe `right-6`? 
So it doesn't look bad when there is no scroll, and does not overlap when there is.